pyre/extra_tests/snippets/exec_into_a_plain_dict_survives_collection.py

```
@@ -0,0 +1,62 @@
+        b = Idx(i % 5 + 3)
+        acc = acc + len(seq[a:b]) + len(lst[a:b]) + len(tup[a:b])
+        i = i + 1
+    assert acc == 450000, acc
+    assert call(lambda: 1) == 1
+
+
+main()
+'''
+
+code = compile(SOURCE, "<foreign globals>", "exec")
+exec(code, {"__name__": "__main__"})
```

Ruff (0.16.2)

[error] Use of `exec` detected

(S102)